Revert the test to not have a little endian condition
as the corruption is now detected (probably) correctly:
https://sqlite.org/forum/forumpost/316effd37e4d1f07

diff --git a/test/fts3corrupt4.test b/test/fts3corrupt4.test
index a23a24b..e0e4c99 100644
--- a/test/fts3corrupt4.test
+++ b/test/fts3corrupt4.test
@@ -4403,31 +4403,12 @@ do_catchsql_test 25.5 {
     INSERT INTO t1( a ) SELECT randomblob(3000) FROM t2 ;
 } {0 {}}
 
-if {$tcl_platform(byteOrder)=="littleEndian"} {
-  # The SQLITE_CORRUPT error depends on the specific random byte
-  # sequence generated by SQLite's PRNG.  But the SQLite PRNG
-  # uses ChaCha20, which generates a different byte sequence on
-  # big-endian and little-endian platforms.  The SQLITE_CORRUPT
-  # error only comes up when the pseudo-random byte sequence is
-  # the one generated on little-endian platforms.
-  #
-  # See Forum thread:
-  # https://sqlite.org/forum/forumpost/b5f89d813babfd88
-  #
-  do_catchsql_test 25.6a {
-    INSERT INTO t1(t1) SELECT x FROM t2;
-  } {1 {database disk image is malformed}}
-  do_catchsql_test 25.6b {
-    INSERT INTO t1(t1) SELECT x FROM t2;
-  } {1 {database disk image is malformed}}
-} else {
-  do_catchsql_test 25.6a {
-    INSERT INTO t1(t1) SELECT x FROM t2;
-  } {0 {}}
-  do_catchsql_test 25.6b {
-    INSERT INTO t1(t1) SELECT x FROM t2;
-  } {0 {}}
-}
+do_catchsql_test 25.6a {
+  INSERT INTO t1(t1) SELECT x FROM t2;
+} {1 {database disk image is malformed}}
+do_catchsql_test 25.6b {
+  INSERT INTO t1(t1) SELECT x FROM t2;
+} {1 {database disk image is malformed}}
 
 
 #-------------------------------------------------------------------------
